### PR TITLE
[Root] Adding README.md for CloudSqlImportOptmization

### PR DESCRIPTION
**Pull request by ldanielmadariaga**
Note: bash commands will be replaced to Gradle once the tool is added

### CODE REVIEW

File: CloudSqlImportOptimization/README.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+TIP: For more information on the limitations of using **--tab** see
+[mysqldump's docs.](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_tab)
+
+In case runnning the exporter locally is a no-go in your environment here are
+the steps necessary for you to produce a MySQL dump that will be compatible with
+the importer:
+
+*   Get your database's schema by calling `mysqldump --no-data`
+*   Ensure the database schema follows the following naming convention:
+    <database_name>_schema.sql
```
Comment (EricBeach) [inline CloudSqlImportOptimization/README.md]: `...`

File: CloudSqlImportOptimization/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+*   Dump your database's data by calling `mysqldump --order-by-primary`
+*   For each table in your database create a new text file <table_name>.txt and
+    copy it's data into said file.
+*   The importer assumes that all .txt files in the dump directory are data
+    files please ensure there are no extra text files in your dump
+    directory. ### Importing your MySQL dump into CloudSQL
```
Comment (EricBeach) [inline CloudSqlImportOptimization/README.md]: need a break

File: CloudSqlImportOptimization/README.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+WARNING: We don't read the Option File's connection settings so you'll still
+need to pass these in via cmd arguments.
+
+If your MySQL implementation does not support Option Files or you just prefer to
+do so you can provide your credentials by using
+**--do_not_use_credentials_file** alongside the **--user** and **--password**
+options. For example:
+
+        $ ./cloudsql_importer.py <database-name> <mysql-dump-folder> --host=<cloud-sql-ip> --do-not-use-credentials-file --user=<user> --password=<password>
```
Comment (EricBeach) [inline CloudSqlImportOptimization/README.md]: foo=bar \
foo1=bar1 \

File: CloudSqlImportOptimization/README.md (hunk 1 above)
Comment (ldanielmadariaga) [reply CloudSqlImportOptimization/README.md]: The tool needn't run in GCE but I can make a note if need be, thoughts?

Comment (EricBeach) [reply CloudSqlImportOptimization/README.md]: correct; my point was if you us GCE, you need a good strong instance

File: CloudSqlImportOptimization/README.md (hunk 2 above)
Comment (ldanielmadariaga) [reply CloudSqlImportOptimization/README.md]: Done

File: CloudSqlImportOptimization/README.md (hunk 3 above)
Comment (ldanielmadariaga) [reply CloudSqlImportOptimization/README.md]: Done

**Pull request merged**